src/components/MessageList.js

```diff
@@ -5,6 +5,7 @@ import MyMessageBubble from "./MyMessageBubble";
 export default class MessageList extends React.Component {
   constructor() {
     super();
+    this.date = new Date().toLocaleDateString()
```

Comment:
Is this necessary ? (if you think it is then you can tell me and we will discuss)